examples/24.authWIthTradingKey.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
```

Comment:
Name of file should not have capital 'I'

Reply:
fixed in 88a91b3